numpy/lib/tests/test_type_check.py

```diff
@@ -98,21 +98,53 @@ def test_real(self):
         y = np.random.rand(10,)
         assert_array_equal(y, np.real(y))
 
+        y = np.array(1)
+        out = np.real(y)
+        assert_array_equal(y, out)
+        assert_(isinstance(out, np.ndarray))
+
+        y = 1
+        assert_equal(y, np.real(y))
```

Comment:
This doesn't actually check that the scalar is returned - all of these tests would have passed before this patch, I think.